### PR TITLE
Ability to limit available fiat currencies

### PR DESCRIPTION
**Pull request by YauhenHuchok**
Allow server admin to limit the number of available fiat currencies

allowed_currencies_demo

### CODE REVIEW

**Comment by dni** [comment]
nice PR! thanks.

can you maybe add it to the admin_ui frontend aswell? so it editable there

**Comment by arcbtc** [comment]
Agreed. This is really useful, but x10 more if you could add to the admin frontend 🙏

**Comment by YauhenHuchok** [comment]
I will add support for Admin UI

**Comment by YauhenHuchok** [comment]
Added allowed currencies selector to Server tab of Admin UI
image

File: lnbits/settings.py [hunk 1]
```diff
@@ -43,6 +43,10 @@ class UsersSettings(LNbitsSettings):
     lnbits_allowed_users: List[str] = Field(default=[])
 
 
+class AllowedCurrencies(LNbitsSettings):
+    lnbits_allowed_currencies: List[str] = Field(default=[])
```
Comment (motorina0) [inline lnbits/settings.py:47]: Can you please move `lnbits_allowed_currencies` under `ThemesSettings`.

**Review by motorina0 — APPROVED** [review]

**Review by motorina0 — APPROVED** [review]

**Comment by callebtc** [comment]
cACK

**Review by callebtc — APPROVED** [review]

**Pull request merged**